backend/src/api/paths/auth.ts

```diff
@@ -0,0 +1,450 @@
+import type { Context, Next } from "hono"
+
+import { createRoute, OpenAPIHono, z } from "@hono/zod-openapi"
+import {
+  generateAuthenticationOptions,
+  generateRegistrationOptions,
+  verifyAuthenticationResponse,
+  verifyRegistrationResponse,
+} from "@simplewebauthn/server"
+import { jwt, sign } from "hono/jwt"
+
+import { prisma } from "../../db"
+import * as schemas from "../components/schemas/auth"
+
+// 環境設定
+const config = {
+  jwtSecret: process.env.JWT_SECRET,
+  origin: process.env.RP_ORIGIN || "https://localhost:5173",
+  rpId: process.env.RP_ID || "localhost",
+  rpName: process.env.RP_NAME || "Shuiro",
+}
+
+if (!config.jwtSecret) {
+  throw new Error("JWT_SECRET is not set")
+}
+
+// 汎用エラーメッセージ
```

Comment:
IMO: 関数の説明等はJSDocで書いていただけると、エディタでホバーした際に説明が表示されるので助かります

```suggestion
/**
 * 汎用エラーメッセージ
 */
```